webapp/javascript/pages/IntroPages/SignIn/index.tsx

```diff
@@ -46,7 +47,9 @@ function SignInPage() {
 
       const res = await logIn({ username, password });
       if (res.isOk) {
-        dispatch(loadCurrentUser());
+        history.replace(
+          (location.state as any)?.redir || PAGES.CONTINOUS_SINGLE_VIEW
```

Comment:
this fixes https://github.com/pyroscope-io/pyroscope/issues/1037 right?

Reply:
Will it work properly with BaseURL? For example, if redir is null, this will redirect to `/`, not `BaseURL`, right?

Reply:
No, history shouldn't be aware. Wondering why tests are passing.